packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/ContentTypeBuilderNav.tsx

```diff
@@ -1,4 +1,5 @@
-import { Fragment } from 'react';
+import { useEffect, useRef, useCallback, Fragment } from "react";
```

Comment:
This import is useless, inside the component you can use the hooks with this syntax `React.useEffect`